swagger.yaml

```diff
@@ -1,42 +1,244 @@
 swagger: "2.0"
+
 info:
   title: dp-search-reindex-api
   description: "Provides details about search reindex jobs and enables creation and running of them"
   version: 1.0.0
   license:
     name: "Open Government Licence v3.0"
     url: "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
+
 basePath: "/v1"
+
 schemes:
   - http
+
 tags:
-  - name: "hello"
+  - name: "Public"
+
 paths:
-  /hello:
+
+  /jobs/{id}:
     get:
       tags:
-        - hello
-      summary: Example hello world endpoint
-      description: Returns an example payload.
+        - "Public"
+      summary: Get a specific job.
+      description: Get the specific search reindex job that has the id given in the path.
+      parameters:
+        - $ref: '#/parameters/id'
       produces:
         - application/json
       responses:
         200:
           description: OK
           schema:
-            $ref: "#/definitions/HelloResponse"
+            $ref: "#/definitions/Job"
+        500:
+          $ref: '#/responses/InternalError'
+
+  /jobs:
+    get:
+      tags:
+        - "Public"
+      summary: Get a list of jobs.
+      description: Get a list of search reindex jobs that meet the criteria specified by the four query parameters.
+      parameters:
+        - $ref: '#/parameters/limit'
+        - $ref: '#/parameters/offset'
+        - $ref: '#/parameters/sort'
+        - $ref: '#/parameters/state'
+      produces:
+        - application/json
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: "#/definitions/Jobs"
+        500:
+          $ref: '#/responses/InternalError'
+    post:
+      tags:
+        - "Public"
+      summary: Create a new search reindex job.
+      description: Post details of a new search reindex job and receive the new Job object created in response.
+      parameters:
+        - $ref: '#/parameters/job'
+      produces:
+        - application/json
+      responses:
+        200:
+          description: OK
```

Comment:
A brief description would be good

Reply:
Yes it should be 201 when creating a resource and concur a more detailed description is helpful here

Reply:
accepted, changed